src/www/ui_tests/api/Controllers/JobControllerTest.php

```diff
@@ -132,6 +132,31 @@ private function getResponseJson($response)
     return json_decode($response->getBody()->getContents(), true);
   }
 
+  /**
+   * Generate array of users
+   * @param array $userIds User ids to be generated
+   * @return array[]
+   */
+  private function getUsers($userIds)
+  {
+    $userArray = array();
+    foreach ($userIds as $userId) {
+      if ($userId == 2) {
+        $accessLevel = PLUGIN_DB_ADMIN;
+      } elseif ($userId > 2 && $userId <= 4) {
+        $accessLevel = PLUGIN_DB_WRITE;
+      } elseif ($userId == 5) {
+        $accessLevel = PLUGIN_DB_READ;
+      } else {
+        continue;
+      }
+      $user = new User($userId, "user$userId", "User $userId",
+        "user$userId@example.com", $accessLevel, 2, 4, "");
+      $userArray[] = $user->getArray();
+    }
+    return $userArray;
+  }
+
```

Comment:
Is this function still needed?

Reply:
yes will be modifying this in the upcoming PR to have the all jobs thing for admin. As mentioned in the issue.

Reply:
I don't know. Without this the tests were failing last time I Updated the PR